lms/djangoapps/certificates/api.py

```diff
@@ -172,6 +173,14 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
         forced_grade - a string indicating to replace grade parameter. if present grading
                        will be skipped.
     """
+    users_queryset = list_with_level(course, u'beta')
```

Comment:
Also course default value is `None`, what if the course object isn't being to this function from where the certificate generation is being triggered?